conanfile.py

```diff
@@ -25,6 +25,10 @@ class ImguiSFML(ConanFile):
     * imconfig_install_folder: None or String
       Use 'None' if you want this build to include imconfig.h file,
       otherwise use anything else.
+
+    * imgui_revision: None or String
+      Tag or branch of ImGui repository that should be patched with
```

Comment:
"should be pached" => "should be used"?

Reply:
As per current implementation, ImGui-SFML providing ImGui with SFML patches. I wanted to make it clear for users, but I don't have a strong opinion on that, because it's basically just wording. I will change that, of course, if you find the second one more suitable.

Reply:
There are no "patches" in ImGui's source code. imconfig-SFML.h is an additional header which is included via the standard extension mechanism.